src/components/Pagination/Pagination.jsx

```diff
@@ -0,0 +1,141 @@
+import React from 'react';
+import './Pagination.scss';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+
+class Pagination extends React.Component {
+  state = {
+    maxCountPage: this.props.maxCount,
+  };
```

Comment:
Don't save props values in `state`.  You can access them directly from `this.props` object.